### PR TITLE
Add experimental Sequel dataset-based query

### PR DESCRIPTION
**Pull request by careo**

### CODE REVIEW

File: lib/praxis-mapper/query/sequel.rb [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      # Executes this SQL statement.
+      # Does not perform any validation of the statement before execution.
+      #
+      # @return [Array] result-set
+      def _execute(ds=nil)
+        Praxis::Mapper.logger.debug "SQL:\n#{self.describe}\n"
+        self.statistics[:datastore_interactions] += 1
+        start_time = Time.now
+
+        rows = if @raw_query
```
Comment (blanquer) [inline lib/praxis-mapper/query/sequel.rb]: what happens if there's a @raw_query set, but the "ds" is passed in too?

Comment (careo) [inline lib/praxis-mapper/query/sequel.rb]: The raw query would win. Considering that's only used via `#multi_get` method above, that's not exactly the correct behavior, but shouldn't pose a problem. 

It's also the same behavior you'd see in the existing SQL query if you did a multi-get on a query that had raw sql defined. So at least we're consistent about doing the wrong thing?

Comment (blanquer) [inline lib/praxis-mapper/query/sequel.rb]: sure, sounds good. Might be nice to throw a warning though.

**Comment by blanquer** [comment]
LGTM

**Pull request merged**